runtime/src/bank.rs

```diff
@@ -4242,27 +4245,51 @@ impl Bank {
             .filter_map(|(index, res)| match res {
                 // following are retryable errors
                 Err(TransactionError::AccountInUse) => {
+                    bidirection_reply_service.send_message(
```

Comment:
why do you need to trigger this from two places?

Reply:
Because these types of errors is not tracked in qos_service.rs.

Reply:
oh i see, so they are exhaustive and non-overlapping?

Reply:
Normally yes but still will confirm it while doing final review.